### DISCUSSION TITLE
Downgrading Doesn't Work?

### DISCUSSION THREAD

**Discussion by Su4o** [3kh0/ext-remover#726] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Info :

CHROMEOS_RELEASE_BUILD_TYPE |   | Official Build
-- | -- | --
CHROMEOS_RELEASE_CHROME_MILESTONE |   | 116
CHROMEOS_RELEASE_DESCRIPTION |   | 15509.63.0 (Official Build) stable-channel octopus
CHROMEOS_RELEASE_KEYSET |   | mp-v32
CHROMEOS_RELEASE_NAME |   | Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER |   | 0
CHROMEOS_RELEASE_TRACK |   | stable-channel
CHROMEOS_RELEASE_UNIBUILD |   | 1
CHROMEOS_RELEASE_VERSION |   | 15509.63.0

[CHROMEOS_RELEASE_BUILD_TYPE](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_BUILD_TYPE)
Official Build
[CHROMEOS_RELEASE_CHROME_MILESTONE](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_CHROME_MILESTONE)
116
[CHROMEOS_RELEASE_DESCRIPTION](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_DESCRIPTION)
15509.63.0 (Official Build) stable-channel octopus 
[CHROMEOS_RELEASE_KEYSET](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_KEYSET)
mp-v32
[CHROMEOS_RELEASE_NAME](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_NAME)
Chrome OS
[CHROMEOS_RELEASE_PATCH_NUMBER](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_PATCH_NUMBER)
0
[CHROMEOS_RELEASE_TRACK](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_TRACK)
stable-channel
[CHROMEOS_RELEASE_UNIBUILD](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_UNIBUILD)
1
[CHROMEOS_RELEASE_VERSION](https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/debugd/docs/log_entries.md#CHROMEOS_RELEASE_VERSION)
15509.63.0
----------------------------------------------------------------------------------
So there's probably no fix to this but i thought that one of you guys would know. 

When downgrading, i'm appeared at this message "out of date update" (something like that)

I've tried to use the latest versions on my computer on chrome100.dev but it still doesn't work 

![image](https://github.com/3kh0/ext-remover/assets/127699325/1c652b38-1c32-4a96-90a2-9b2973c313b5)

anyone knows another way to downgrade or fix? there probably isn't tho.

**Comment by AshtonDavies** [comment, marked as answer]
Patched in v116
This has already been discussed

**Comment by abditube1345** [comment]
i downgrade my school chromebook to v109 and not updating

**Comment by abditube1345** [comment]
but i cannot im stuck on 119
